charts/k8s-watcher/templates/configmap-daemon.yaml

```diff
@@ -19,6 +19,11 @@ data:
     [[outputs.http]]
       url = "https://app.komodor.com/metrics-collector/api/v1/collect"
       content_encoding = "gzip"
+      {{- if .Values.proxy.https }}
+      http_proxy_url = "${KOMOKW_HTTPS_PROXY_URL}"
+      {{- else if .Values.proxy.http }}
+      http_proxy_url = "${KOMOKW_HTTP_PROXY_URL}"
+      {{- end }}
```

Comment:
did you check it ? 
i dont think it works inside a string value

Reply:
Yes..it works..the whole file is defined like this

Reply:
even the ifs ?

Reply:
yes...tested with --set proxy.http="something" and --set proxy.https="sdad"